source-oracle/replication.go

```diff
@@ -53,6 +53,13 @@ func (db *oracleDatabase) ReplicationStream(ctx context.Context, startCursor str
 		return nil, fmt.Errorf("set NLS_TIMESTAMP_TZ_FORMAT: %w", err)
 	}
 
+	// Logminer cannot run on PDB instances, so we switch to the CDB
```

Comment:
Can `switchToCDB` be used here?

Reply:
we use a separate connection for replication and that method runs on `db.conn`, whereas this one is run on a new and separate connection